Reconstruction of ServiceControl menu (#5437)
Particular/docs.particular.net: servicecontrol/how.md

  classDef ServiceControlInstance fill:#A84198,stroke:#92117E,color:#FFFFFF,stroke-width:4px
  classDef Plugin fill:#00729C,stroke:#000,color:#ddd

  class ServiceControlError ServiceControlInstance
  class ServiceControlAudit ServiceControlInstance
  class ServiceControlMonitoring ServiceControlInstance
  class Watchers Endpoints
  class Plugins Plugin
  class Audit Plugin
  class Error Plugin
  class Monitoring Plugin
```

### Error instances

In NServiceBus, [recoverability](/nservicebus/recoverability/) is a very important feature. It enables automatic retries and continuity within a system, as failed messages will be moved aside. Those messages however still contain business data that needs to be looked after and eventually be processed.

NServiceBus will move messages it cannot process to an [error queue](/nservicebus/recoverability/#fault-handling). This is where ServiceControl comes in as it will consume these messages. This means it will pick up the message and store it inside an internal database. For the internal database it uses RavenDb. After it is stored in the database, it is made available to ServicePulse and ServiceInsight for visualization and retries.

> NOTE: It is recommended to not provide end-users with the ability to retry messages. The message could fail again and end up in ServiceControl again anyway. It could be even more problemetic when many messages are retried during a peak in message processing. This will result in even more messages to be processed by an endpoint, causing valid messages to be delayed even longer.

Reviewer: ```suggestion
NOTE: It is recommended to not provide end-users with the ability to retry messages. The message could fail again and end up in ServiceControl again anyway. It could be even more problematic when many messages are retried during a peak in message processing. This will result in even more messages being processed by an endpoint, causing valid messages to be delayed even longer.
```

Find out more about [failed messages](/servicepulse/intro-failed-messages.md) in ServicePulse.

### Audit instances

> To provide the ability for ServiceInsight to visualize the message flow through the system, it needs to have access to every single message that has been successfully processed by the system. This is enabled within endpoints by [enabling auditing](/nservicebus/operations/auditing.md), followed by ServiceControl that consumes these messages and stores them in its internal database.

Reviewer: ```suggestion
To enable ServiceInsight to visualize the message flow through the system, it needs to have access to every single message that has been successfully processed by the system. This requires endpoints to [enable auditing](/nservicebus/operations/auditing.md). ServiceControl consumes these messages and stores them in its internal database.
```

> ServiceInsight will retrieve the data from ServiceControl via the HTTP API and use header information (that NServiceBus added during processing and sending it to the audit queue) to figure out which message caused other messages to be send, including which sagas were accessed when the [SagaAudit plugin](/nservicebus/sagas/saga-audit.md) is configured with an endpoint.

Reviewer: ```suggestion
ServiceInsight will retrieve the data from ServiceControl via the HTTP API and use header information (which was added by NServiceBus during message processing) to figure out which message caused other messages to be sent, including which sagas were accessed when the [SagaAudit plugin](/nservicebus/sagas/saga-audit.md) is configured in an endpoint.
```
